widget/src/index.tsx

```diff
@@ -211,10 +225,18 @@ async function init() {
         return;
     }
 
+    const callTitle = getCallTitle();
+
+    // Setting the base URL if present, in case MM is running under a subpath.
+    if (window.basename) {
+        Client4.setUrl(window.basename);
+    }
+
     // initialize some basic state.
     await Promise.all([
         getMe()(store.dispatch, store.getState),
         getMyPreferences()(store.dispatch, store.getState),
+        getMyTeams()(store.dispatch, store.getState),
         getChannelAction(channelID)(store.dispatch, store.getState),
```

Comment:
I believe all of these, and the ones below, can be replaced with
```suggestion
        store.dispatch(getMe())
        store.dispatch(getMyPreferences()),
        store.dispatch(getMyTeams()),
        store.dispatch(getChannelAction(channelID)),
```
Which is a bit more idiomatic. But otherwise it's the same, so I don't have a strong opinion either way.

Reply:
Thanks, looks cleaner for sure!

Reply:
Little update on this: since they'd be causing a merge conflict with what I am currently working on for recordings I'll be addressing this as part of that work.